api/v1beta1/cloudstackcluster_types.go

```diff
@@ -131,7 +132,7 @@ type CloudStackClusterStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
-
+// +k8s:conversion-gen=false
```

Comment:
```suggestion
//+k8s:conversion-gen=false
```
Let's remove the space after // to be consistent